ruleset/sca/amazon/cis_amazon_linux_2022.yml

```diff
@@ -0,0 +1,5084 @@
+# Security Configuration Assessment
+# CIS Checks for Amazon Linux 2022
+# Copyright (C) 2022, Wazuh Inc.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation
+#
+# Based on:
+# Center for Internet Security Amazon Linux 2022 Benchmark v1.0.0 - 06-02-2022
+
+policy:
+  id: "cis_amazon_linux_2022"
+  file: "cis_amazon_linux_2022.yml"
+  name: "CIS Benchmark for Amazon Linux 2022"
+  description: "This document provides prescriptive guidance for establishing a secure configuration posture for Amazon Linux 2022 systems running on x86 and x64 platforms. This document was tested against Amazon Linux 2022."
+  references:
+    - https://www.cisecurity.org/cis-benchmarks/
+
+requirements:
+  title: "Check Amazon Linux platform."
+  description: "Requirements for running the policy against Amazon Linux 2022."
+  condition: any
+  rules:
+    - "f:/etc/system-release -> r:^Amazon && r:release 2022"
+
+variables:
+  $sshd_file: /etc/ssh/sshd_config
+
+checks:
+  ############################################################
+  # 1.1 Filesystem Configuration
+  ############################################################
+
+  # 1.1.1.1 Ensure mounting of cramfs filesystems is disabled.
+  - id: 31000
+    title: "Ensure mounting of cramfs filesystems is disabled."
+    description: "The cramfs filesystem type is a compressed read-only Linux filesystem embedded in small footprint systems. A cramfs image can be used without having to first decompress the image."
+    rationale: "Removing support for unneeded filesystem types reduces the local attack surface of the server. If this filesystem type is not needed, disable it."
+    remediation: "Edit or create the file /etc/modprobe.d/cis.conf and add a line that reads install cramfs /bin/false and a line that reads blacklist cramfs. Run the following command to unload the cramfs module: modprobe -r cramfs"
+    compliance:
+      - cis: ["1.1.1.1"]
+      - cis_csc_v8: ["4.8"]
+      - cis_csc_v7: ["9.2"]
+      - nist_sp_800-53: ["CM-7"]
+      - cmmc_v2.0: ["CM.L2-3.4.7", "CM.L2-3.4.8", "SC.L2-3.13.6"]
+      - pci_dss_3.2.1: ["1.1.6", "1.2.1", "2.2.2", "2.2.5"]
+      - pci_dss_4.0: ["1.2.5", "2.2.4", "6.4.1"]
+      - soc_2: ["CC6.3", "CC6.6"]
+    condition: all
+    rules:
+      - "c:modprobe -n -v cramfs -> r:install /bin/false|Module cramfs not found"
+      - "not c:lsmod -> r:cramfs"
+      - 'd:/etc/modprobe.d -> r:\.*.conf -> r:blacklist\scramfs'
+
+  # 1.1.1.2 Ensure mounting of squashfs filesystems is disabled.
+  - id: 31001
+    title: "Ensure mounting of squashfs filesystems is disabled."
+    description: "The squashfs filesystem type is a compressed read-only Linux filesystem embedded in small footprint systems (similar to cramfs). A squashfs image can be used without having to first decompress the image."
```

Comment:
The description is not the same as the pdf.

Reply:
Solved